pkg/hints/math_hints.go

```diff
@@ -151,3 +151,118 @@ func sqrt(ids IdsManager, vm *VirtualMachine) error {
 	ids.Insert("root", NewMaybeRelocatableFelt(root_felt), vm)
 	return nil
 }
+
+// Implements hint:
+//
+//	from starkware.cairo.common.math_utils import as_int
+//	# Correctness check.
+//	value = as_int(ids.value, PRIME) % PRIME
+//	assert value < ids.UPPER_BOUND, f'{value} is outside of the range [0, 2**250).'
+//	# Calculation for the assertion.
+//	ids.high, ids.low = divmod(ids.value, ids.SHIFT)
+func Assert250Bit(ids IdsManager, vm *VirtualMachine, constants *map[string]Felt) error {
+	const UPPER_BOUND = "starkware.cairo.common.math.assert_250_bit.UPPER_BOUND"
+	const SHIFT = "starkware.cairo.common.math.assert_250_bit.SHIFT"
+
+	if constants == nil {
+		return errors.New("Called Assert250Bit with a nil constants map")
+	}
```

Comment:
I think this check could be redundant, this method is only called during `ExecuteHint` which is only called during `Step` and so on, the we should be able to trust that the constants reference is not nil  as we also trust that the vm reference is not nil

Reply:
Makes sense, removed here [476ab88](https://github.com/lambdaclass/cairo-vm_in_go/pull/233/commits/476ab88dbab2305ceedd3209486f06d14270616a)